README.md

```diff
@@ -1,62 +1,100 @@
-# Spring Web MVC Logging
+# Logbook
 
-[![Build Status](https://img.shields.io/travis/zalando/spring-web-logging.svg)](https://travis-ci.org/zalando/spring-web-logging)
-[![Coverage Status](https://img.shields.io/coveralls/zalando/spring-web-logging.svg)](https://coveralls.io/r/zalando/spring-web-logging)
-[![Release](https://img.shields.io/github/release/zalando/spring-web-logging.svg)](https://github.com/zalando/spring-web-logging/releases)
-[![Maven Central](https://img.shields.io/maven-central/v/org.zalando/spring-web-logging.svg)](https://maven-badges.herokuapp.com/maven-central/org.zalando/spring-web-logging)
+[![Build Status](https://img.shields.io/travis/zalando/logbook.svg)](https://travis-ci.org/zalando/logbook)
+[![Coverage Status](https://img.shields.io/coveralls/zalando/logbook.svg)](https://coveralls.io/r/zalando/logbook)
+[![Release](https://img.shields.io/github/release/zalando/logbook.svg)](https://github.com/zalando/logbook/releases)
+[![Maven Central](https://img.shields.io/maven-central/v/org.zalando/logbook.svg)](https://maven-badges.herokuapp.com/maven-central/org.zalando/logbook)
 
-Spring Web MVC request and response logging (including payload).
+Servlet 3.0 filter for request and response logging (including payload).
 
-# Usage
+## Dependency
 
-You have to register the ``LoggingFilter`` as a ``Filter`` of your dispatcher filter chain.
-
-```java
-@Bean
-public FilterRegistrationBean loggingFilter() {
-    final LoggingFilter filter = new LoggingFilter();
-    final FilterRegistrationBean registration = new FilterRegistrationBean();
-    registration.setFilter(filter);
-    registration.setDispatcherTypes(EnumSet.allOf(DispatcherType.class));
-    registration.addUrlPatterns("/api/*");
-    registration.setOrder(filter.getOrder());
-    return registration;
-}
+```xml
+<dependency>
+    <groupId>org.zalando</groupId>
+    <artifactId>logbook</artifactId>
+    <version>${logbook.version}</version>
+</dependency>
 ```
 
-# Customization
+## Usage
+
+You have to register the `LogbookFilter` as a `Filter` in your filter chain.
+
+Either in your `web.xml` file:
+
+```xml
+<filter>
+    <filter-name>LogbookFilter</filter-name>
+    <filter-class>org.zalando.logbook.LogbookFilter</filter-class>
+</filter>
+<filter-mapping>
+    <filter-name>LogbookFilter</filter-name>
+    <url-pattern>/*</url-pattern>
+    <dispatcher>FORWARD</dispatcher>

```

Comment:
Would we actually need the once per request logic if the filter is not registered for FORWARD / INCLUDE?